src/modules/shared/pages/innovations/innovation-advanced-search-card.component.ts

```diff
@@ -5,6 +5,12 @@ import { InnovationStatusEnum } from '@modules/stores/innovation';
 import { InnovationSupportStatusEnum } from '@modules/stores/innovation';
 import { InnovationGroupedStatusEnum, InnovationSectionEnum } from '@modules/stores/innovation/innovation.enums';
 
+export enum InnovationSupportCloseReasonEnum {
+  ARCHIVE = 'ARCHIVE',
+  STOP_SHARE = 'STOP_SHARE',
+  SUPPORT_COMPLETE = 'SUPPORT_COMPLETE'
+}
+
```

Comment:
I would put this in `innovation.enums.ts`

Reply:
Makes sense, done